assets/app/styles/_core.less

```diff
@@ -18,6 +19,10 @@ html, body {
     background-color: @body-bg;
     padding-bottom: @grid-gutter-width * 2;
     .flex(@columns: 1);
+    h1 {
+      overflow: hidden;
+      text-overflow: ellipsis;
```

Comment:
I saw the screenshot. Are we putting the full name as a title so it's available on hover?